Lib/asyncio/base_events.py

```diff
@@ -404,8 +404,11 @@ def __init__(self):
         # A weak set of all asynchronous generators that are
         # being iterated by the loop.
         self._asyncgens = weakref.WeakSet()
+
```

Comment:
Please remove this empty line; it wasn't there to visually group all stuff related to asyncgens.

Reply:
Oh okay, I just thought it would be useful to group the shutdown flags visually. I'll remove it.

Reply:
Yeah, sorry for the nitpicking :)

Reply:
@1st1:
> Yeah, sorry for the nitpicking :)

No problem, I don't mind nitpicking. I can definitely understand why you guys are tight on the formatting changes, it helps to keep the code more organized. (: